### ISSUE TITLE
Publish to Nuget failed for release 2.4.0

### ISSUE THREAD

**Issue by TommyN** [firebase/firebase-admin-dotnet#351]
The 2.4.0 release failed in the publish to Nuget step.

See: https://github.com/firebase/firebase-admin-dotnet/actions/runs/5350795917/jobs/9703890826

```
Run ./.github/scripts/publish_package.sh
info : Pushing FirebaseAdmin.2.4.0.nupkg to 'https://www.nuget.org/api/v2/package'...
info :   PUT https://www.nuget.org/api/v2/package/
info :   Forbidden https://www.nuget.org/api/v2/package/ 241ms
error: Response status code does not indicate success: 403 (The specified API key is invalid, has expired, or does not have permission to access the specified package.).
Error: Process completed with exit code 1.
```

**Comment by TommyN** [comment]
cc: @lahirumaramba

**Comment by TommyN** [comment]
Seems to have been fixed here:
https://github.com/firebase/firebase-admin-dotnet/actions/runs/5535725575

**Comment by lahirumaramba** [comment]
Hey folks, we were able to fix the issue and `v2.4.0` should be available on Nuget now. Thanks for your patience on this!
